### DISCUSSION TITLE
Github Copilot Master Research

### DISCUSSION THREAD

**Discussion by Momo-tech** [community/community#58747] [Copilot Conversations]
Hi Github Copilot Team and Users,

I planning to do a research on the effectiveness of AI Code completion tools in Web Development.
Is there any way i can make the participants use github copilot without purchasing a plan for the 30-60 minute experiment.

I tried live share which let the user not work with copilot without having the extension themself. 

Thanks for the responses in advance

**Comment by the-officialjosh** [comment]
Considering that most students are eligible for the student pack, which includes access to GitHub Copilot, it could be advantageous for you to engage students as participants or collaborate with them using this information. However, it's important to note that I'm not certain whether there is a trial available specifically for GitHub Copilot.
